Sim/SimPapas/src/PapasExportParticlesTool.h

```diff
@@ -0,0 +1,39 @@
+#ifndef SIM_PAPASEXPORTPARTICLES_H
+#define SIM_PAPASEXPORTPARTICLES_H
+
+#include "FWCore/DataHandle.h"
+#include "GaudiAlg/GaudiTool.h"
+#include "SimPapas/IPapasTool.h"
+#include "datamodel/ParticleCollection.h"
+
+#include <string>
+
+/** @class PapasExportParticlesTool Sim/SimPapas/src/PapasExportParticlesTool.h PapasExportParticlesTool.h
+ *
+ *  Takes papasparticles stored in the papas::Event and converts to MCParticles.
+ *  @author A.J. Robson
+ */
+class PapasExportParticlesTool : public GaudiTool, virtual public IPapasTool {
+public:
+  /// Constructor.
+  PapasExportParticlesTool(const std::string& aType, const std::string& aName, const IInterface* aParent);
+  /// Initialize.
+  virtual StatusCode initialize();
+  /// create outputs
+  virtual StatusCode createOutputStructures();
+  /// Execute:
+  virtual StatusCode run(papas::Event& pevent);
+  /// Clear:
+  virtual StatusCode clear() { return SUCCESS; };
+  /// Finalize.
+  virtual StatusCode finalize();
+
+private:
+  /// handle for reconstructed particles collection
+  DataHandle<fcc::ParticleCollection> m_particlesHandle{"recparticles", Gaudi::DataHandle::Writer, this};
+  /// Particles type to be exported
+  Gaudi::Property<std::string> m_particleSubtype{this, "particleSubtype", "", "particle subtype"};
+  fcc::ParticleCollection* m_particles;  /// mcparticles collection
```

Comment:
It is needed in the case when papas cannot complete the simulation in order to ensure that the exporter always has an object to export (even if its empy). It will be addressed as part of issue #220